src/deb/control/templates

```diff
@@ -0,0 +1,39 @@
+Template: [[artifactId]]/rhino_db_host
```

Comment:
Please add an explanatory comment here for what this file does and how it interacts with pre/post scripts.